docs/notes/2.22.x.md

```diff
@@ -108,6 +108,16 @@ The default version of `semgrep` used by the `pants.backends.experimental.tool.s
 
 [The `pants.backend.shell.lint.shfmt` backend](https://www.pantsbuild.org/2.22/docs/shell#shfmt-autoformatter) now uses shfmt version 3.8.0 by default.
 
+#### Trufflehog
```

Comment:
@benjyw please let me know if this is the right place for this! I added based on the suggestion of a failed CI check